src/Codeception/Lib/Generator/Test.php

```diff
@@ -31,12 +30,17 @@ public function testSomeFeature()
 }
 EOF;
 
-    protected $testerTemplate = <<<EOF
-    /**
-     * @var \{{actorClass}}
-     */
+    protected $testerLegacyTemplate = <<<EOF
```

Comment:
I don't like this name - it calls PHP 7.3 legacy.

Reply:
Lets prepare for the future 😁